lib/logstash/plugin_mixins/common.rb

```diff
@@ -0,0 +1,90 @@
+module LogStash
+  module PluginMixins
+    module SchemaRegistry
```

Comment:
In case some other plugin wants to implement a "schema registry", we should namespace this `KafkaAvroSchemaRegistry` or something similarly descriptive.